### PR TITLE
fix(plasma-ui): [Picker] Fix flicking and fake onChange in Safari 14.5 and bellow

### PR DESCRIPTION
**Pull request by Yakutoc**
### Проблема 

В Safari iOS 14.5 и ниже наблюдаются следующее

- ложные срабатывания onChange 
- flicking/мерцания/дёргание дочерних элементов компонента

Важное замечание - проблемы **взаимосвязаны**.

### Было:

Screen Recording 2023-04-05 at 17 16 50

### Стало: 

Screen Recording 2023-04-05 at 17 22 33

------

:baby_chick: Download canary assets:

[colors--canary.464.4626740634.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.464.4626740634.xml)  
[color_sberHealth_ios-swift--canary.464.4626740634.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.464.4626740634.swift)  
[color_sberHealth_kotlin--canary.464.4626740634.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.464.4626740634.kt)  
[color_sberHealth_react-native--canary.464.4626740634.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.464.4626740634.ts)  
[color_sbermarket_business_ios-swift--canary.464.4626740634.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.464.4626740634.swift)  
[color_sbermarket_business_kotlin--canary.464.4626740634.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.464.4626740634.kt)  
[color_sbermarket_business_react-native--canary.464.4626740634.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.464.4626740634.ts)  
[color_sbermarket_ios-swift--canary.464.4626740634.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.464.4626740634.swift)  
[color_sbermarket_kotlin--canary.464.4626740634.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.464.4626740634.kt)  
[color_sbermarket_metro_ios-swift--canary.464.4626740634.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.464.4626740634.swift)  
[color_sbermarket_metro_kotlin--canary.464.4626740634.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.464.4626740634.kt)  
[color_sbermarket_metro_react-native--canary.464.4626740634.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.464.4626740634.ts)  
[color_sbermarket_react-native--canary.464.4626740634.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.464.4626740634.ts)  
[color_sbermarket_selgros_ios-swift--canary.464.4626740634.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.464.4626740634.swift)  
[color_sbermarket_selgros_kotlin--canary.464.4626740634.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.464.4626740634.kt)  
[color_sbermarket_selgros_react-native--canary.464.4626740634.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.464.4626740634.ts)  
[color_sbermarket_vinlab_ios-swift--canary.464.4626740634.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_vinlab_ios-swift--canary.464.4626740634.swift)  
[color_sbermarket_vinlab_kotlin--canary.464.4626740634.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_vinlab_kotlin--canary.464.4626740634.kt)  
[color_sbermarket_vinlab_react-native--canary.464.4626740634.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_vinlab_react-native--canary.464.4626740634.ts)  
[color_sberprime_ios-swift--canary.464.4626740634.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.464.4626740634.swift)  
[color_sberprime_kotlin--canary.464.4626740634.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.464.4626740634.kt)  
[color_sberprime_react-native--canary.464.4626740634.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.464.4626740634.ts)  
[PlasmaTokensColor--canary.464.4626740634.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.464.4626740634.swift)  
[shadow_sbermarket_react-native--canary.464.4626740634.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.464.4626740634.ts)  
[typo_mage_ios-swift--canary.464.4626740634.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.464.4626740634.swift)  
[typo_mage_kotlin--canary.464.4626740634.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.464.4626740634.kt)  
[typo_mage_react-native--canary.464.4626740634.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.464.4626740634.ts)  
[typo_plasma_ios-swift--canary.464.4626740634.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.464.4626740634.swift)  
[typo_plasma_kotlin--canary.464.4626740634.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.464.4626740634.kt)  
[typo_plasma_react-native--canary.464.4626740634.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.464.4626740634.ts)  
[typo_ruler_ios-swift--canary.464.4626740634.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.464.4626740634.swift)  
[typo_ruler_kotlin--canary.464.4626740634.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.464.4626740634.kt)  
[typo_ruler_react-native--canary.464.4626740634.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.464.4626740634.ts)  
[typo_sage_ios-swift--canary.464.4626740634.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.464.4626740634.swift)  
[typo_sage_kotlin--canary.464.4626740634.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.464.4626740634.kt)  
[typo_sage_react-native--canary.464.4626740634.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.464.4626740634.ts)  
[typo_sbermarket_ios-swift--canary.464.4626740634.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.464.4626740634.swift)  
[typo_sbermarket_kotlin--canary.464.4626740634.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.464.4626740634.kt)  
[typo_sbermarket_react-native--canary.464.4626740634.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.464.4626740634.ts)  
[typo_soulmate_ios-swift--canary.464.4626740634.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.464.4626740634.swift)  
[typo_soulmate_kotlin--canary.464.4626740634.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.464.4626740634.kt)  
[typo_soulmate_react-native--canary.464.4626740634.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.464.4626740634.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-temple@1.149.0-canary.464.4626740634.0
  npm install @salutejs/plasma-ui@1.181.0-canary.464.4626740634.0
  # or 
  yarn add @salutejs/plasma-temple@1.149.0-canary.464.4626740634.0
  yarn add @salutejs/plasma-ui@1.181.0-canary.464.4626740634.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-464/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-464/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-464/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-464/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-464/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-464/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-464/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-464/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-464/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-464/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-464/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-464/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-464/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-464/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-464/

File: packages/plasma-ui/src/components/Pickers/PickerItem.tsx [hunk 1]
```diff
@@ -120,6 +127,7 @@ export const PickerItem = ({
     onItemClick,
     autofocus,
     disabled,
+    disableScrollSnapAlign,
```
Comment (neretin-trike) [inline packages/plasma-ui/src/components/Pickers/PickerItem.tsx:130]: тут нужно `disableScrollSnapAlign = false` как в TimerPicker?

**Review by neretin-trike — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-464/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-464/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-464/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-464/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-464/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

File: website/plasma-ui-docs/docs/components/Pickers.mdx [hunk 2]
```diff
@@ -87,3 +87,27 @@ export function App() {
 
 При этом, с активированным экранным считывающим устройством (screen reader),
 происходит объявление переключаемых опций.
+
+## Проблемы в Safari iOS 14.5 и ниже
```
Comment (Yeti-or) [inline website/plasma-ui-docs/docs/components/Pickers.mdx:91]: 👍

**Review by Yeti-or — APPROVED** [review]

**Comment by Yeti-or** [comment]
Надо ещё в сторибук добавить переключение этого свойства, иначе я не могу протестировать

File: packages/plasma-ui/src/components/Pickers/PickerItem.tsx (hunk 1 above)
Comment (Yakutoc) [reply packages/plasma-ui/src/components/Pickers/PickerItem.tsx:130]: @neretin-trike Это свойство прилетает сверху, где или будет выставленное значение или **false** по-умолчанию. На этом уровне ИМХО нет особого смысла ставить **default** значение. 

Или ты про что? )

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-464/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-464/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-464/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-464/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-464/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-464/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-464/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-464/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-464/

File: packages/plasma-ui/src/components/Pickers/PickerItem.tsx (hunk 1 above)
Comment (neretin-trike) [reply packages/plasma-ui/src/components/Pickers/PickerItem.tsx:134]: да, я про это, оке 👌

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-464/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-464/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-464/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-464/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-464/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-464/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Pull request merged**